File: 2A_2/cpp/funkcje02.cpp
/*
 * funkcje02.cpp
 * 
 */

#include <iostream>
using namespace std;

int a = 0;  // zmienna globalna
int b = 0;

void sumuj(){
	int a = 5;
	int b = 5;
	int suma = a + b;
	cout << "Suma: " << suma;
}

int main(int argc, char **argv)
{
	cout << "Podaj liczby: " << endl;
	cin >> a;
	cin >> b;
	
	sumuj();  // wywołanie funkcji
	
	return 0;
}

File: 2A_2/python/funkcje02.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  funkcje02.py
#  

a = int(input("Podaj liczbę 1: "))  # zmienna globalna
b = int(input("Podaj liczbę 2: "))  # zmienna globalna

def sumuj(a, b):
    suma = a + b
    print("Suma:", suma)


def sumuj2():
    suma = a + b
    print("Suma:", suma)


def main(args):
    l1 = int(input("Podaj liczbę 3: "))  # zmienna lokalna
    l2 = int(input("Podaj liczbę 4: "))
    sumuj(a, b)
    sumuj2()
    return 0


if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))
